ui/src/app/views/pipeline/add/pipeline.add.component.ts

```diff
@@ -53,4 +64,18 @@ export class PipelineAddComponent {
         });
 
     }
+
+    goToProject(): void {
+        this._router.navigate(['/project', this.project.key], {queryParams: {tab: 'pipelines'}});
+    }
+
+    importPipeline() {
+        this.loadingCreate = true;
+        this._pipStore.importPipeline(this.project.key, this.pipToImport)
```

Comment:
Possible to add a popup warning when a pipeline with the same name exist?

Reply:
forceUpdate deleted